.github/workflows/maven.yml

```diff
@@ -0,0 +1,20 @@
+name: Maven CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Build with Maven
+        run: mvn clean install --file pom.xml
```

Comment:
Do we need `install`?  For Jenkins jobs, we do not need to install the package into Jenkins local repo. `package` would be better?

Reply:
No, we don't, good catch